packages/core/resources/icons/icons.json

```diff
@@ -2840,5 +2840,26 @@
     "tags": "sidebar, layout, spreadsheet",
     "group": "table",
     "content": "\\e778"
+  },
+  {
+    "name": "Tick circle",
+    "className": "pt-icon-tick-circle",
+    "tags": "tick mark, sign, ok, approved, success",
```

Comment:
doesn't even need to contain "tick," cuz that's part of the name. "mark" should be enough